File: aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/TypeTemplateService.java
package com.aisile.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.aisile.pojo.TbTypeTemplate;
import com.aisile.pojo.entity.PageResult;
import com.aisile.pojogroup.Specification;

public interface TypeTemplateService {

	List<TbTypeTemplate> findAll();

	PageResult findPage(int page, int rows);

	PageResult findPage(TbTypeTemplate tbTypeTemplate, int pageNum,int pageSize);

	void add(TbTypeTemplate tbTypeTemplate);

	TbTypeTemplate findOne(Long id);

	void update(TbTypeTemplate tbTypeTemplate);

	void delete(Long[] ids);

	public List<Map> findSpecList(Long id);

}

File: aisile-manager-web/src/main/webapp/js/service/typeTemplateService.js
		 //品牌服务层
		 app.service('typeTemplateService',function($http){
			    	//读取列表数据绑定到表单中
			    	this.findAll=function(){
			    		return $http.get('../typeTemplate/findAll.do');		
			    	}
			    	//分页
			    	this.findPage=function(page,rows){
			    		return $http.get('../typeTemplate/findPage.do?page='+page+'&rows='+rows);		
			    	}
			    	//搜索
			    	this.search=function(page,rows,searchEntity){
			    		return $http.post('../typeTemplate/search.do?page='+page+"&rows="+rows,searchEntity);
			    	}
			    	//保存
			    	this.save = function(methodName,entity){
			    		return $http.post('../typeTemplate/'+ methodName +'.do',entity);
			    	}
			    	//回显
			    	this.findOne = function(id){
			    		return $http.get('../typeTemplate/findOne.do?id='+id);
			    	}
			    	//删除
			    	this.dele=function(selectIds){
			    		return $http.get('../typeTemplate/delete.do?ids='+selectIds);
			    	}
			    	//查询规格列表
			    	this.findSpecList=function(id){
			    		return $http.get('../typeTemplate/findSpecList.do?id='+id);
			    	}
		 }); 